presto-main/src/test/java/io/prestosql/metadata/TestInformationSchemaMetadata.java

```diff
@@ -217,6 +218,36 @@ public void testInformationSchemaPredicatePushdownOnCatalogWiseTables()
         assertFalse(result.isPresent());
     }
 
+    @Test
+    public void testInformationSchemaPredicatePushdownForEmptyNames()
+    {
+        TransactionId transactionId = transactionManager.beginTransaction(false);
+
+        Constraint constraint = new Constraint(TupleDomain.fromFixedValues(
+                ImmutableMap.of(
+                        new InformationSchemaColumnHandle("table_name"), new NullableValue(VARCHAR, Slices.utf8Slice("")))));
+        ConnectorSession session = createNewSession(transactionId);
+        ConnectorMetadata metadata = new InformationSchemaMetadata("test_catalog", this.metadata);
+        InformationSchemaTableHandle tableHandle = (InformationSchemaTableHandle)
+                metadata.getTableHandle(session, new SchemaTableName("information_schema", "tables"));
+        tableHandle = metadata.applyFilter(session, tableHandle, constraint)
+                .map(ConstraintApplicationResult::getHandle)
+                .map(InformationSchemaTableHandle.class::cast)
+                .orElseThrow(AssertionError::new);
+        assertTrue(tableHandle.getPrefixes().isEmpty());
+
+        tableHandle = (InformationSchemaTableHandle)
+                metadata.getTableHandle(session, new SchemaTableName("information_schema", "tables"));
+        constraint = new Constraint(TupleDomain.fromFixedValues(
+                ImmutableMap.of(
+                        new InformationSchemaColumnHandle("table_schema"), new NullableValue(VARCHAR, Slices.utf8Slice("")))));
+        tableHandle = metadata.applyFilter(session, tableHandle, constraint)
+                .map(ConstraintApplicationResult::getHandle)
+                .map(InformationSchemaTableHandle.class::cast)
+                .orElseThrow(AssertionError::new);
+        assertTrue(tableHandle.getPrefixes().isEmpty());
```

Comment:
can you please extract an assertion method and call it twice: `assertApplyFiterReturnsEmptyPrefixes(             ImmutableMap.of(
                        new InformationSchemaColumnHandle("table_schema"), new NullableValue(VARCHAR, Slices.utf8Slice(""))));`? and same for `table_name`